spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskDeleteService.java

```diff
@@ -262,44 +264,7 @@ public void deleteTaskExecutions(Set<Long> taskExecutionIds) {
 		auditData.put("Deleted Job Execution IDs", StringUtils.collectionToDelimitedString(jobExecutionIds, ", "));
 
 		if (!jobExecutionIds.isEmpty()) {
-			final Set<Long> stepExecutionIds = dataflowJobExecutionDao.findStepExecutionIds(jobExecutionIds);
```

Comment:
Ahh, ok. This is just applying the same chunking treatment to the step executions+contexts that was previously added to this method in https://github.com/spring-cloud/spring-cloud-dataflow/pull/4723.